README.md

```diff
@@ -185,11 +185,10 @@ To publish locally, you'll need to do the following...
 ####Sign In
 
 To publish locally, you'll need an account.
-* Go to [http://localhost:3000/account](http://localhost:3000/account)
-* Click ``Join Webmaker`` and complete the process, you can use a fake email
-* When you've created your account, click ``Set permanent password instead``
-  * This lets you authenticate your account without needing email
-* Go back to Thimble and Log In with your new account
+* Go to [http://localhost:3000](http://localhost:3000)
```

Comment:
@humphd  Yeah I can!, I'll do it when I get home later tonight. So just indicate the buttons on the top right corner. "create an account" and "Sign in"  instead of having the link ?